commit 1fe4ed0eb91f0dde5987ad0f41a2c5d02198b080
Author: Alex Wilson <alex.wilson@joyent.com>
Date:   2019-06-07T23:00:45+00:00 (4 months ago)
    
    MANTA-4328 zkstream missed wakeups lead to binder sadness
    MANTA-4331 muppet stuck after ZK election

diff --git a/package.json b/package.json
index f5d072c..7980839 100644
--- a/package.json
+++ b/package.json
@@ -16,7 +16,7 @@
         "once": "1.4.0",
         "vasync": "2.2.0",
         "verror": "1.10.0",
-        "zkstream": "0.11.4"
+        "zkstream": "0.11.6"
     },
     "devDependencies": {
         "nodeunit": "0.11.2",
